custom_components/zaptec/sensor.py

```diff
@@ -38,27 +38,50 @@ def _update_from_zaptec(self) -> None:
         self._attr_available = True
 
 
-class ZaptecChargeSensor(ZaptecSensor):
+class ZaptecSensorTranslate(ZaptecSensor):
+    """Sensor with strings intended for traslations."""
```

Comment:
```suggestion
    """Sensor with strings intended for translations."""
```
Maybe also include a description for why we need this, ala
```
    Several native Zaptec keys use capital letters. 
    Translation keys need to be lower case in order to pass hassfest.
```